qemu/tests/cfg/dump_guest_core.cfg

```diff
@@ -0,0 +1,22 @@
+- dump_guest_core:
+    type = dump_guest_core
+    virt_test_type = qemu
+    pre_command = "ulimit -c unlimited"
+    pre_command += "&& echo "/var/core.%p" > /proc/sys/kernel/core_pattern"
+    trigger_core_dump_command = "kill -s SIGSEGV %s"
+    gdb_command_file = "/home/gdb_command"
+    crash_script = "/home/crash.cmd"
+    vmcore_file = "/tmp/vmcore"
+    core_file = "/var/core.%s"
+    gdb_command = "gdb %s --command=${gdb_command_file}"
+    crash_cmd = "crash -i ${crash_script} /usr/lib/debug/lib/modules/%s/vmlinux %s"
```

Comment:
And here: `crash_cmd = "crash -i ${crash_script} /usr/lib/debug/lib/modules/%s/vmlinux ${vmcore_file}"`

Reply:
Okay, thanks for your help, the code has been updated.Will update test result later.